test/webgl/buffer.spec.js

```diff
@@ -97,3 +97,50 @@ test('WebGL#Buffer setData/subData', t => {
 
   t.end();
 });
+
+test('WebGL#Buffer getData', t => {
+  const {gl2} = fixture;
+  if (!gl2) {
+    t.comment('WebGL2 not available, skipping tests');
+    t.end();
+    return;
+  }
+
+  let data = new Float32Array([1, 2, 3]);
+  let buffer = new Buffer(gl2, {data});
+
+  let receivedData = buffer.getData({});
```

Comment:
Can we add default `({...}={})` parameter to `getData` so that we don't need to use empty braces here?